src/node/serialization.cc

```diff
@@ -102,7 +102,7 @@ class NodeIndexer : public AttrVisitor {
       for (const auto& kv : n->data) {
         MakeIndex(const_cast<Object*>(kv.second.get()));
       }
-    } else {
+    } else if (!node->IsInstance<StringObj>()) {
```

Comment:
@tqchen Could you please please take a look at the serialization of String here? I am not sure why the size of the String I loaded back is always 0.

Reply:
Indeed, we should design a special serialization mechanism for String. A quick idea is to just print out the string content, we can also think about base64 encoding.  I can try to take a stab tomorrow.

Reply:
I saved it separately to handle this for now